test/powershell/Modules/Microsoft.PowerShell.Security/FileCatalog.Tests.ps1

```diff
@@ -409,6 +409,13 @@ Describe "Test suite for NewFileCatalogAndTestFileCatalogCmdlets" -Tags "CI" {
             $result = Test-FileCatalog -Path $testDataPath\UserConfigProv\ -CatalogFilePath $script:catalogPath -FilesToSkip "*.psd1","UserConfigProviderModVersion2.psm1","*ModVersion1.schema.mof"
             $result | Should -Be "Valid"
         }
+
+        It "TestCatalogWhatIfForNewFileCatalog"{
```

Comment:
```suggestion
        It "New-FileCatalog -WhatIf does not create file" {
```

Reply:
The rest of the file has the smooshed test titles... do you think we should move away from what the rest of the file does, @rjmholt?

Reply:
Generally speaking in newer tests we opt for _not_ smooshing them. Probably this test file should have a followup PR to make the test titles more readable across the whole file when we're done here. 🙂